index.js

```diff
@@ -99,10 +168,51 @@
   in the terminal
  @Test - Make a function call of `trackGallonsUsed(5)`
 */
+function trackGallonsUsed(gallons) {
+    let consoleLog = []
+
+    while (gallons > 0) {
+        if (gallons === 1) {
+            console.log(`${gallons} Gallon Remaining`)
+            consoleLog.push(`${gallons} Gallon Remaining`)
+            gallons--
+        } else {
+            console.log(`${gallons} Gallons Remaining`)
+            consoleLog.push(`${gallons} Gallons Remaining`)
+            gallons--
+        }
+    }
+    return consoleLog
+}
 
 
 /* 
  @Challenge 10 - Write a function named `getHighScore` that finds the highest value in an array of scores
  @Example - Sending the function a value of [1999,2020,3080,1111] will result in 3080
  @Test - Write a console.log that shows the value of `getHighScore([1999,2020,3080,1111])`
 */
+function getHighScore(listOfScores){
+    let highScore = 0
+
+    listOfScores.forEach(currentScore => {
+        if(highScore < currentScore){
+            highScore = currentScore
+        }
+    })
+    console.log(highScore)
+    return highScore
+}
+
```

Comment:
Looks like an extra `getHighScore` reference sitting on line 206. No harm in it but making note since it is there.